README.md

```diff
@@ -23,6 +23,15 @@ All addresses are stored in `.txt` files with CIDRs, where each range is on a ne
 - IPv6: https://raw.githubusercontent.com/lord-alfred/ipranges/main/google/ipv6.txt
 - IPv6 (merged): https://raw.githubusercontent.com/lord-alfred/ipranges/main/google/ipv6_merged.txt
 
+### Google (GoogleBot)
```

Comment:
I may be wrong, and I apologize if I am.

However, when I did a test merge between `google/ipv{4,6}.txt` and `googlebot/ipv{4,6}.txt`, the end result was identical to `google/ipv{4,6}.txt`.

Because of this, I think Google includes the GoogleBot addresses in its other stuff as well, so I kept GoogleBot in the section header for Google and added the note about needing to specifically allow the GoogleBot addresses if desired.

Reply:
Thank you for that observation. I think it would be better if we leave google content as it is, but make a separate section with bot addresses. Anyway, since the ranges overlap (partially) - it's worth leaving google as it is, so that the scripts where it is used now don't break.

Reply:
Sounds good to me! It is done.

Reply:
No, I meant leave google as it is (don't edit anything there), but add only googlebot next to it.

Reply:
I have tried again. I apologize if I got it wrong again, but it seemed like you wanted me to change the Google section title back to what it was ("Google (Cloud & GoogleBot)").